addons/html_builder/static/src/builder/core_builder_action_plugin.js

```diff
@@ -70,21 +103,16 @@ export const coreBuilderActions = {
         },
     },
     styleAction: {
-        getValue: ({ editingElement, param: styleName }) => {
-            const customStyle = styleMap[styleName];
-            if (customStyle) {
-                return customStyle.getValue(editingElement);
-            } else {
-                return getComputedStyle(editingElement).getPropertyValue(styleName);
-            }
-        },
-        apply: ({ editingElement, param: styleName, value }) => {
-            const customStyle = styleMap[styleName];
-            if (customStyle) {
-                customStyle?.apply(editingElement, value);
-            } else {
-                editingElement.style.setProperty(styleName, value);
+        getValue: getStyleActionValue,
+        apply: applyStyleAction,
+        clean: applyStyleAction,
```

Comment:
As if we put "0" as a value for example, we still want this value to be applied in the style